src/components/PermanentConfirmationDialog/PermanentConfirmationDialog.jsx

```diff
@@ -0,0 +1,32 @@
+import React, { PureComponent } from 'react';
+import Button from 'material-ui/Button';
+import Dialog from 'material-ui/Dialog';
+import DialogActions from 'material-ui/Dialog/DialogActions';
+import DialogContent from 'material-ui/Dialog/DialogContent';
+import DialogContentText from 'material-ui/Dialog/DialogContentText';
+import DialogTitle from 'material-ui/Dialog/DialogTitle';
+
+class PermanentConfirmationDialog extends PureComponent {
+  render() {
+    const { open, handleOnClick, submitMutation } = this.props;
+
+    return (
+      <Dialog open={open} onRequestClose={() => handleOnClick(false)}>
+        <DialogTitle>Are you sure you want to delete member?</DialogTitle>
```

Comment:
We should move this title to a prop of this component so we can reuse this dialog for other components.